.rubocop_todo.yml

```diff
@@ -13,9 +13,9 @@ Lint/HandleExceptions:
 
 # Offense count: 19
 Metrics/AbcSize:
-  Max: 56
+  Max: 100
```

Comment:
Thats a big difference. Cant you add a Rubocop exception on the specific methods?